assets/css/glimesh/components/sidebar.scss

```diff
@@ -0,0 +1,50 @@
+.sidebar {
+    margin: 0;
+    padding: 0;
+    width: 300px;
+    position: absolute;
+    height: 100%;
+    overflow: auto;
+}
+
+.sidebar a {
+    display: block;
+    color: white;
+    padding: 16px;
+    text-decoration: none;
+}
+
+.sidebar a.active {
+    background-color: #1b2e4b;
+    color: #009688;
+}
+
+.sidebar a:hover:not(.active) {
+    background-color: #1b2e4b;
+    color: #009688;
+}
+
+div.content {
+    margin-left: 300px;
+    padding: 1px 16px;
+    height: 100%;
+}
+
+/* On screens that are less than 700px wide, make the sidebar into a topbar */
+@media screen and (max-width: 700px) {
+  .sidebar {
+    width: 100%;
+    height: auto;
+    position: relative;
+  }
+  .sidebar a {float: left;}
```

Comment:
Consider using `flexbox` for laying out elements to one side or another, `float` is largely considered obsolete for this, and is really only meant for things like floating text around an image in an article.
https://stackoverflow.com/a/9777248